modules/apps/portal/portal-upload-test-util/src/main/java/com/liferay/portal/upload/test/util/UploadTestUtil.java

```diff
@@ -16,19 +16,31 @@
 
 import com.liferay.portal.kernel.test.ReflectionTestUtil;
 import com.liferay.portal.kernel.upload.FileItem;
+import com.liferay.portal.kernel.upload.UploadPortletRequest;
 import com.liferay.portal.kernel.upload.UploadServletRequest;
 import com.liferay.portal.kernel.util.PortalUtil;
+import com.liferay.portal.upload.UploadPortletRequestImpl;
 
 import java.util.List;
 import java.util.Map;
 
+import javax.portlet.PortletRequest;
+
 import javax.servlet.http.HttpServletRequest;
 
 /**
  * @author Jiefeng Wu
  */
 public class UploadTestUtil {
 
+	public static UploadPortletRequest createUploadPortletRequest(
```

Comment:
The reason I added this method is, I would like to centralize the logic of using new UploadPortletRequestImpl() in the test.
Later when I moved `UploadPortletRequestImpl()` to modules, I will use `BundleWiring Bundles ` to get the logic of Impl as Dante advised, and I will replace new UploadPortletRequestImpl() to using Bundle in this code